src/ar_node_worker.erl

```diff
@@ -445,6 +446,18 @@ process_new_block(#{ height := Height, cumulative_diff := CDiff } = StateIn, New
 			none
 	end.
 
+log_missing_txs(TXIDs, ReconstructedTXs) ->
+	case length(TXIDs) of
+		L when L > length(ReconstructedTXs) ->
+			MissingTXIDs = TXIDs -- lists:map(fun(TX) -> TX#tx.id end, ReconstructedTXs),
```

Comment:
We calculate the missing TXIDs here. Wouldn't it make sense to log if MissingTXIDs is not empty? Parent case doesn't make sense to me.

Reply:
The parent case is just a quicker way to see if txs are missing. If yes, I subrtact the sets to log them.

Reply:
What if:
TXIDs: [a, b, c]
ReconstructedTXs: [b, c, d]

Or:
TXIDs: [a, b, c]
ReconstructedTXs: [e, f, g, h, i, j, k]

Reply:
Btw, I would suggest using `lists:member/2` together with `lists:all/2`. For larger lists it would make sense to use the `sets` module.

Reply:
`TXIDs` can't be smaller by construction.

Reply:
> Btw, I would suggest using lists:member/2 together with lists:all/2

What is your concern?